patientsearch/src/js/components/PatientListTable.js

```diff
@@ -125,10 +133,16 @@ export default function PatientListTable(props) {
     SortArrow: forwardRef((props, ref) => <ArrowDownward {...props} ref={ref} color="primary" />),
```

Comment:
Not sure which line is actually relevant, but would be very helpful on first page load, if the arrow indicating sort order were already in place on the **Last Accessed** column. At the moment, the "sorting arrow" only appears on a column, after clicking it.

Reply:
thank you, ivan. should be fixed.